indigo_api/tests/data_migrations/test_paras_to_hcontainer.py

```diff
@@ -0,0 +1,296 @@
+                    <p>The City reserves the right to call upon the owner or his or her agent of the fuel burning equipment to have the emissions from such fuel burning equipment evaluated at his or her own expense as may be required by the authorised official.</p>
+                  </content>
+                </paragraph>
+                <hcontainer id="schedule1.crossheading-0" name="crossheading">
+                  <heading>Insulation of chimneys:</heading>
+                </hcontainer>
+                <paragraph id="schedule1.paragraph0">
+                  <content>
+                    <p>All fuel-burning equipment using Heavy Fuel Oil or other liquid fuels with a sulphur content equal to or greater than 2.5 % by weight must be fitted with a fully insulated chimney using either a 25mm air gap or mineral wool insulation to prevent the formation of acid smut. Such chimneys must be maintained in a good state of repair at all times.</p>
+                  </content>
+                </paragraph>
+                <hcontainer id="schedule1.crossheading-1" name="crossheading">
+                  <heading>Wood-fired pizza ovens and other solid fuel combustion equipment:</heading>
+                </hcontainer>
+                <paragraph id="schedule1.paragraph1">
+                  <content>
+                    <p>Wood-fired pizza ovens and other solid fuel combustion equipment shall be fitted with induced draft fans at the discretion of the authorised official.</p>
+                  </content>
+                </paragraph>
+              </hcontainer>
+            </mainBody>
+          </doc>
+        </component>
+      </components>
+    </akomaNtoso>
+        """)
+            migration.migrate_document(doc)
+            doc.refresh_xml()
+            self.assertMultiLineEqual(
+                """<akomaNtoso xmlns="http://www.akomantoso.org/2.0">
+      <act>
+        <body/>
+      </act>
+      <components>
+        <component id="component-schedule1">
+          <doc name="schedule1">
+            <meta/>
+            <mainBody>
+              <hcontainer id="schedule1" name="schedule">
+                <subheading>Subheading</subheading>
+                <hcontainer id="schedule1.hcontainer_1">
+                  <content>
+                    <p>schedule 1</p>
+                  </content>
+                </hcontainer>
+              </hcontainer>
+            </mainBody>
+          </doc>
+        </component>
+        <component id="component-schedule2">
+          <doc name="schedule2">
+            <meta/>
+            <mainBody>
+              <hcontainer id="schedule2" name="schedule">
+                <heading>Schedule 2</heading>
+                <subheading>REPEAL OR AMENDMENT OF LAWS</subheading>
+                <hcontainer id="schedule2.crossheading-0" name="crossheading">
+                  <heading>(Section 45(1))</heading>
+                </hcontainer>
+                <hcontainer id="schedule2.hcontainer_1">
```

Comment:
```suggestion
                <hcontainer id="schedule2.hcontainer_2">
```

This should be hcontainer_2, because there is already an hcontainer before it (as a sibling), even though it's really a crossheading. I must change slaw to generate hcontainer ids for crossheadings.